.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
+<!-- Deleting this template may result in your issue being closed prematurely. -->
```

Comment:
I'd like to propose using positive language here.
```suggestion
<!-- Please fill out the following template, which will help other contributors address your issue. -->
```

Reply:
I'm okay with this, but it doesn't address the issue of people deleting the template entirely.  Would prefacing it with "please do not delete this template" be okay with you?

Reply:
@bunder2015 I think it does address the issue of people deleting the template, because deleting the template is not filling it out.

@kpande By "positive" I mean let's tell folks what we want them to do, rather than what we don't want them to do.

Reply:
the whole template is already telling people what to do though, right? this is just one part of it, which is being put in because presently there is confusion and frustration from users who submitted an issue or PR without the template when their issue is closed. we should make it clearer that it will be closed.

Reply:
I like @ahrens's wording here. There's value in explaining upfront why a template is provided and that filling it out will help their PR make it though the process.

Reply:
I not understand why https://github.com/openzfs/openzfs/ have no "Issues" page, maybe to block tickets spam too?